cmd/usage/usage.go

```diff
@@ -182,20 +186,13 @@ func outputCSV(resp ybmclient.BillingUsageData, filename string) error {
 		row := []string{dataPoint.GetStartDate()}
 
 		for _, dimension := range resp.GetDimensions() {
-			var found bool
 			for _, point := range dimension.DataPoints {
-				if point.StartDate == dataPoint.StartDate {
```

Comment:
This is why we need unit tests :)

Reply:
Added unit tests for different scenarios, please check.